### DISCUSSION TITLE
How to Static Build

### DISCUSSION THREAD

**Discussion by djm858** [yermak/AudioBookConverter#485] [Q&A]
I'm fairly new to Linux Mint 20.3. I'm trying to install AudioBookConverter and it suggests the static build. I downloaded  AudioBookConverter-static-binaries-6.1.2.tar.gz and extracted it in the Downloads folder. I have no idea what to do from there, or if I need to do something else entirely. Googling "how to install or run static build programs" but it's not helping.

Can anyone assist?

**Comment by iconoclasthero** [comment]
your best bet is to put it in your local bin folder, e.g, mine is ~/bin/ and then you can launch it from the command line by e.g., 
`$ ~/bin/AudioBookConverter/bin/AudioBookConverter` 

You can also put ./AudioBookConverter/bin/AudioBookConvert in /bin but you have to put ./AudioBookConverter/lib/* (i.e. _the contents_  of it) in /lib.  The latter worked on ubuntu 23.04, but it's kinda messy since you you have all that stuff sitting in the system folder.  You could also put both in /opt or something and then launch it from there...
